### PR TITLE
Extend policy probationary period until May 31st, 2021.

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Other

**Required reviews:** 
1

**What this does:**
Extends policy probationary period until May 31st, 2021.

**Why it's needed:**
Prevent CLI CI failures and give additional development runway before permitting long-lived and high-value policies.

### CODE REVIEW

**Review by derekpierre — APPROVED** [review]
🎸 - Do we want to do a newsfagment (done for #2585 ) or since this is only temporary until we have a discussion, no need for a newsfragment...? It does affect app developers.

I guess after an in-depth discussion it is more likely to extend the probationary period than reduce 😅 , so a newsfragment this time would not be a bad thing.

**Comment by KPrasch** [comment]
Yes, let's do a newsfragment, it's exactly the kind of thing it's good at!  Standby.

**Comment by derekpierre** [comment]
One nitpick - May has 31 days 😅 , we should do the 31st. After that merge at will.

File: nucypher/config/constants.py [hunk 1]
```diff
@@ -73,4 +73,4 @@
 NUCYPHER_EVENTS_THROTTLE_MAX_BLOCKS = 'NUCYPHER_EVENTS_THROTTLE_MAX_BLOCKS'
 
 # Probationary period (see #2353, #2584)
-END_OF_POLICIES_PROBATIONARY_PERIOD = MayaDT.from_iso8601('2021-04-30T23:59:59.0Z')
+END_OF_POLICIES_PROBATIONARY_PERIOD = MayaDT.from_iso8601('2021-05-30T23:59:59.0Z')
```
Comment (derekpierre) [inline nucypher/config/constants.py:76]:
```suggestion
END_OF_POLICIES_PROBATIONARY_PERIOD = MayaDT.from_iso8601('2021-05-31T23:59:59.0Z')
```

File: newsfragments/2656.misc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Extends policy probationary period until May 30th, 2021.  No policies may be created on the network beyond this date.
```
Comment (derekpierre) [inline newsfragments/2656.misc.rst:1]:
```suggestion
Extends policy probationary period until May 31st, 2021.  No policies may be created on the network beyond this date.
```

**Pull request merged**